spec/std/system_spec.cr

```diff
@@ -18,4 +18,12 @@ describe System do
       cpu_count.should eq(shell_cpus)
     end
   end
+
+  describe "login" do
+    it "returns String when found" do
+      shell_login = `echo $USER`.strip
```

Comment:
According to travis doc there is a `$USER` env var on all builds: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
It just states that the value may not always be `travis`.